src/headless/HeadlessTools.cpp

```diff
@@ -0,0 +1,29 @@
+#include "HeadlessApps.h"
+#include "HeadlessPluginLayerProxy.h"
+
+namespace Surge
+{
+namespace Headless
+{
+
+SurgeSynthesizer* createSurge()
```

Comment:
Maybe a singleton and `getSurgeInstance()`?

Reply:
I’d thought about that but I wanted to be able to simulate a host which was running 5 surge instances in one process. But some management utilities for the pointer would be useful I agree